src/main/resources/application-jrd-router.yaml

```diff
@@ -4,45 +4,59 @@ start-route: direct:JRD
 archival-route: direct:JrdArchival
 archival-path: azure-blob://${azure.storage.account-name}/jud-ref-data-archive
 active-blob-path: azure-blob://${azure.storage.account-name}/jud-ref-data
-archival-file-names: Personal.csv,Appointments.csv
+archival-file-names: Personal-test.csv,Appointments-test.csv
 archival-cred: credentials=#credsreg&operation=updateBlockBlob
```

Comment:
is this for testing only? this needs to be updated before prod release

Reply:
done